rsconf/package_data/dev/db/000.yml.jinja

```diff
@@ -64,9 +64,6 @@ default:
           on_calendar: "1 0"
           bash_script: |
             echo doing that thing
-
```

Comment:
needs one blank line after a section that begins with `|`

Reply:
361965b